packages/gatsby-plugin-sitemap/src/__tests__/gatsby-node.js

```diff
@@ -1,21 +1,30 @@
-jest.mock(`fs`)
+jest.mock(`sitemap`, () => {
+  return {
+    simpleSitemapAndIndex: jest.fn(),
+  }
+})
 
-const fs = require(`fs`)
 const path = require(`path`)
 const sitemap = require(`sitemap`)
-
 const { onPostBuild } = require(`../gatsby-node`)
-import * as internals from "../internals"
+
 const pathPrefix = ``
 
+const reporter = {
+  verbose: jest.fn(),
+  panic: jest.fn().mockImplementation(err => {
+    console.error(err)
+  }),
+}
```

Comment:
Best to not show errors in console during tests
```suggestion
const reporter = {
  verbose: jest.fn(),
  panic: jest.fn(),
}
```

Reply:
Oops, I think this was leftover from debugging, good catch.